lms/envs/common.py

```diff
@@ -2569,6 +2569,7 @@
         "level_of_education",
         "mailing_address",
         "requires_parental_consent",
+        "account_privacy",
```

Comment:
I am moving this to the `shareable_fields` list instead since the clients need to know whether other users' profiles are limited or not.